app/templates/src/main/webapp/scripts/components/auth/services/_password.service.js

```diff
@@ -5,3 +5,15 @@ angular.module('<%=angularAppName%>')
         return $resource('api/account/change_password', {}, {
         });
     });
+
+angular.module('jhipsterApp')
```

Comment:
jhipsterApp should be <%=angularAppName%>

Maybe you should consider just chaining the 3 calls to factory().